.github/workflows/ci-test.yml

```diff
@@ -120,15 +121,10 @@ jobs:
     timeout-minutes: 70
     continue-on-error: false
     needs: prepare
-    #if: ${{ needs.prepare.outputs.test-rust == 'true' }}
+    if: ${{ needs.prepare.outputs.test-rust == 'true' }}
     container:
       image: libra/build_environment:github-1
     steps:
-      - name: stats
-        run: |
-          set x
-          cat /proc/meminfo
```

Comment:
nit - it would be good to keep this and do another stat dump at the end of the job - we can use the delta to tell mem usage at job level and check if there is regression in it.

Reply:
This has been in for a while, we're getting the machine types we're asking for....